conbench/templates/benchmark-result.html

```diff
@@ -185,22 +227,34 @@ <h3>History plot for benchmark '{{ benchmark.display_bmname }}'</h3>
             </li>
           {% endif %}
         {% endif %}
-        <li class="list-group-item list-group-item-primary">Hardware</li>
+        <li class="list-group-item list-group-item-primary">
+          Hardware (checksum is constant between results in history plot above)
+        </li>
+        <li class="list-group-item" style="overflow-y: auto;">
+          <b>checksum</b>
+          <div align="right" style="display:inline-block; float: right;">
+            <code>{{ result.run.hardware.hash }}</code>
+          </div>
+        </li>
         {% for k,v in run.hardware.items() %}
           <li class="list-group-item" style="overflow-y: auto;">
             <b>{{ k }}</b>
-            <div align="right" style="display:inline-block; float: right;">{{ v }}</div>
+            <div align="right" style="display:inline-block; float: right;">
+              <code>{{ v }}</code>
+            </div>
           </li>
         {% endfor %}
-        <li class="list-group-item list-group-item-primary">Context</li>
+        <li class="list-group-item list-group-item-primary">Context (constant between results in history plot above)</li>
```

Comment:
These parentheticals would also fit nicely as (i) tooltips if we prefer

Reply:
yes can do, skipping for now

Reply:
doing that in https://github.com/conbench/conbench/pull/1433